util/util.go

```diff
@@ -45,3 +65,52 @@ func ParseWeekdays(weekdays string) []time.Weekday {
 	}
 	return parsedWeekdays
 }
+
+// ParseTimePeriods takes a comma-separated list of time periods in Kitchen24 format and turns them
+// into a slice of TimePeriods. It ignores any whitespace.
+func ParseTimePeriods(timePeriods string) ([]TimePeriod, error) {
+	parsedTimePeriods := []TimePeriod{}
+
+	for _, tp := range strings.Split(timePeriods, ",") {
+		parts := strings.Split(tp, "-")
+		if len(parts) != 2 {
+			continue
```

Comment:
Isn't this an error condition? Or are you just choosing to ignore it without logging it?

Reply:
Good point. I intended to silently ignore the case when there's only whitespace so that the default case `""` doesn't need any special treatment. I'll provide a fix for when there's a value but doesn't seem to be valid time period, e.g. contains exactly one `-` I guess.